src/test/java/io/jenkins/plugins/artifact_manager_jclouds/s3/MinioIntegrationTest.java

```diff
@@ -64,14 +70,66 @@
     private S3BlobStoreConfig config;
     private AmazonS3 client;    
     private S3BlobStore provider;
-    private JCloudsArtifactManagerFactory artifactManagerFactory;
+    private String prefix;
+    
+    protected String getPrefix() {
+        return prefix;
+    }
     
     @Rule
     public ExpectedException expectedException = ExpectedException.none();
     
     @Rule
     public JenkinsRule jenkinsRule = new JenkinsRule();
     
+    protected ArtifactManagerFactory getArtifactManagerFactory(Boolean deleteArtifacts, Boolean deleteStashes) {
+        return new JCloudsArtifactManagerFactory(new CustomPrefixBlobStoreProvider(provider, getPrefix(), deleteArtifacts, deleteStashes));
+    }
+
+    private static final class CustomPrefixBlobStoreProvider extends BlobStoreProvider {
```

Comment:
I guess this could be split to a top-level class to be shared with the original.

Reply:
Good idea, this has been done